Formula/firefoxpwa.rb

```diff
@@ -19,7 +19,8 @@ class Firefoxpwa < Formula
 
   on_linux do
     depends_on "pkg-config" => :build
-    depends_on "openssl@1.1"
+    depends_on "bzip2" # not used on macOS
```

Comment:
Looks like it doesn't work since pkg-config files don't exist in `bzip2` current release.